src/api/models/user.js

```diff
@@ -0,0 +1,25 @@
+const user = (sequelize, DataTypes) => {
+  const User = sequelize.define('Users', {
+    names: {
```

Comment:
does this represent both the first name and last name? Is there a reason why they aren't separate fields in the DB?

Reply:
it was a mistake according to the documentation I have to store the username, wich is different from user's names